operations/filter_kgraph_discrete_knode_attribute.yml

```diff
@@ -0,0 +1,37 @@
+id: filter_kgraph_discrete_knode_attribute
+subclass_of: filter_kgraph
+name: filter kgraph (discrete knode attribute)
+description: This operation removes kgraph nodes which have a discrete attribute containing the specified value. Node without the given attribute are left alone. Edges connecting to the removed nodes will also be removed.
```

Comment:
Add description about expected behavior in TRAPI 1.1 (attribute type id and the like)